ci/README.md

```diff
@@ -44,13 +44,14 @@ document.
 
 ## Setup & Execution
 
-### Download and install snaps-oo
+### Download and install snaps-orchestration
 
 ```bash
-git clone https://github.com/opnfv/snaps.git
-sudo apt update
-sudo apt install python git python2.7-dev libssl-dev python-pip
-sudo pip install -e snaps
+git clone https://github.com/cablelabs/snaps-orchestration.git
+pip uninstall -y snaps-orchestration
+pip freeze | xargs pip uninstall -y
```

Comment:
I don't know if you want to make everyone uninstall all of their python packages here...